README.md

```diff
@@ -12,6 +12,23 @@ For technical information please refer to the [Gnosis Developer Portal](https://
 
 For support requests, please open up a [bug issue](https://github.com/gnosis/safe-react/issues/new?template=bug-report.md) or reach out via [Discord](https://chat.gnosis-safe.io).
 
+## Transactions
```

Comment:
I suggest we put it in the docs folder as a new document and just link here.

Reply:
Ah, I overlooked the docs folder. I'll change it now